src/vs/workbench/api/common/extHost.protocol.ts

```diff
@@ -1785,6 +1786,7 @@ export interface ExtHostSearchShape {
 	$provideFileSearchResults(handle: number, session: number, query: search.IRawQuery, token: CancellationToken): Promise<search.ISearchCompleteStats>;
 	$provideTextSearchResults(handle: number, session: number, query: search.IRawTextQuery, token: CancellationToken): Promise<search.ISearchCompleteStats>;
 	$clearCache(cacheKey: string): Promise<void>;
+	$doInternalFileSearchWithCustomCallback(query: search.IFileQuery, token: CancellationToken, handleFileMatch: (data: UriComponents[]) => void): Promise<search.ISearchCompleteStats>;
```

Comment:
It looks like this isn't called from the main process, so it shouldn't be in this interface and shouldn't start with `$`.